app/models/user.rb

```diff
@@ -60,6 +60,16 @@ def user_description
     end
   end
 
+  def cohort
+    return early_career_teacher_profile.cohort if early_career_teacher?
+    return mentor_profile.cohort if mentor?
+  end
+
+  def school
+    return early_career_teacher_profile.school if early_career_teacher?
+    return mentor_profile.school if mentor?
+  end
```

Comment:
@BroiSatse is there a better way to delegate this?

Reply:
there's another one for core_induction_programme just above which is basically the same too. it's not great but i didn't want to worry about it too much as some of them will change soon anyway when mentors can have more than one core induction programme

Reply:
I am almost certain a better way to delegate this would involve a polymorphic relationship of some kind, or maybe a `ParticipantProfile` that encompases both ECTs and mentors. Either of those would probably be best done in a separate PR.

Reply:
@ethanmills @kicferk1 The longer i think about this the more it feels like we are missing a Participant model (or PariticpantProfile) maybe MentorProfile and EarlyCareerTeacherProfiles are just a subclass of ParticipantProfile (with STI or MTI?)

Reply:
I am not sure about adding `ParticipantProfile` that is inherited by `EarlyCareerTeacherProfile` and `MentorProfile`. Will the user need to have a link to the `participant_profile`, and that's how we get the common stuff out of them?

I guess it could work, so long as we are sure no teacher will ever simultaneously be a mentor and a mentee 😂 Are we sure of that?

I think we can give it a go and see if things look nicer, but definitely in a separate PR.